modules/olm-injecting-custom-ca.adoc

```diff
@@ -68,3 +68,8 @@ spec:
 <4> `ca-bundle.crt` is required as the config map key.
 <5> `tls-ca-bundle.pem` is required as the config map path.
 <6> Create a `trusted-ca` volume mount.
++
+[NOTE]
+====
+Sometimes deployment of an Operator fails to validate the authority and throws the `x509 certificate signed by unknown authority` error, even after injecting a custom CA by using the subscription of an Operator. In this case, it is recommended to inject the entire CA chain. If the deployment of a Operator still fails then you can set the `mountPath` as `/etc/ssl/certs` for trusted-ca by using the subscription of an Operator.
```

Comment:
`In this case, it is recommended to inject the entire CA chain.`
I'm not sure on the context, but it seems like this sentence needs a prepositional phrase at the end. Like `it is recommended to inject the entire CA chain to the Operator`, or something with the correct term.

Reply:
I confirmed with the bug reporter, we can exclude the whole sentence. 

Addressed

Reply:
Addressed